.github/workflows/doc-builder.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish docs via GitHub Pages
+on:
+  push:
+    branches:
+      - main
+      - dev
+      
+  # Allow the workflow to be triggered also manually.
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - run: pip install mkdocs-material
```

Comment:
Could we pin a specific version of mkdocs-material here? Don't want our docs garbled/comprised/etc. if there's ever an issue in the latest.

Reply:
Fixed. Pinned it to the latest version (8.2.9)